crates/graphql/src/schema/objects/store_creator.rs

```diff
@@ -0,0 +1,32 @@
+use super::prelude::*;
+
+#[derive(Debug, Clone)]
+pub struct StoreCreator {
+    pub store_config_address: String,
+    pub creator_address: String,
+}
+
+#[graphql_object(Context = AppContext)]
```

Comment:
This should just be a `derive(GraphQLObject)`

Reply:
true. maybe but we can switch declaration at that time. sorry should have caught that too.

Reply:
oh yes , I did add one more field to test smth and then removed

Reply:
maybe juniper will have fixed the api for this by that time :crossed_fingers: i missed it as well lol